engine/src/test/resources/org/hibernate/validator/test/internal/xml/containerelementtype/getter-canDeclareContainerElementTypeConstraintsForListContainingArrayType-mapping.xml

```diff
@@ -1,3 +1,9 @@
+<!--
```

Comment:
On the headers of these files, we had left them out intentionally; but then there is no harm by having this small (constant) header, so why not.